src/system/registerHandlebarsHelpers.ts

```diff
@@ -25,7 +26,8 @@ export const registerHandlebarsHelpers = function () {
   });
 
   Handlebars.registerHelper("localizeitemtype", (typeName) => {
-    return getGame().i18n.localize("ITEM.Type" + typeName.titleCase());
+    const itemType: ItemTypeEnum = <ItemTypeEnum>typeName;
```

Comment:
`<ItemTypeEnum>typeName` is the older version on typecasting in ts. Now they recommend `typeName as ItemTypeEnum` but is the typecasting really necessary here?

Reply:
Does not appear to be needed.